website/project/metadata/utils.py

```diff
@@ -27,10 +27,17 @@ def serialize_meta_schemas(meta_schemas):
 
 def serialize_draft_registration(draft, auth=None):
     from website.project.utils import serialize_node  # noqa
+    from website.util import api_v2_url
     from api.base.utils import absolute_reverse
+    from django.core.urlresolvers import NoReverseMatch
 
     node = draft.branched_from
 
+    try:
+        register_url = absolute_reverse('nodes:node-registrations', kwargs={'node_id': node._id, 'version': 'v2'}),
+    except (NoReverseMatch, SyntaxError):  # TODO: fix bug on certain Travis environments
```

Comment:
Another Travis only bug, it's likely because we're in a questionable request context here, some Flask elements, some DRF.

Reply:
>If we need to do a TCI-specific fix, it's better if they go in TCI-specific files. Can it be shoved into a relevant test file instead?

Also, if this util is hit in both v2 and Flask, why doesn't it break during app runtime? Smells like an issue with test case setup